src/mlmc/quantity.py

```diff
@@ -677,15 +682,19 @@ def selection_id(self):
     def get_quantity_storage(self):
         return self
 
-    def samples(self, chunk_spec):
+    def chunks(self, level_id=None, n_samples=None):
```

Comment:
Do we need this? Isn't Storage.chunks enough?

Reply:
It is just a wrapper.